packages/css/src/link/link.scss

```diff
@@ -6,45 +6,41 @@
 @import "../../utils/breakpoint";
 
 .amsterdam-link {
-  color: var(--amsterdam-link-color, blue);
+  color: var(--amsterdam-link-color);
   font-family: var(--amsterdam-paragraph-font-family);
   font-size: var(--amsterdam-link-font-size-narrow);
+  font-weight: var(--amsterdam-link-font-weight);
   line-height: var(--amsterdam-link-line-height);
   outline-offset: var(--amsterdam-link-outline-offset);
-  text-decoration: var(--amsterdam-link-text-decoration, underline);
-  text-decoration-color: var(--amsterdam-link-text-decoration-color, currentColor);
-  text-decoration-thickness: var(--amsterdam-link-text-decoration-thickness, 1px);
+  padding-block: 0.5rem;
+  text-decoration-thickness: var(--amsterdam-link-text-decoration-thickness);
   text-underline-offset: var(--amsterdam-link-text-underline-offset);
 
   &:hover,
-  &:active {
-    color: var(--amsterdam-link-hover-color);
-    text-decoration: var(--amsterdam-link-hover-text-decoration);
-    text-decoration-thickness: var(--amsterdam-link-hover-text-decoration-thickness, 1px);
-  }
-
+  &:active,
   &:focus {
     color: var(--amsterdam-link-focus-color);
-    text-decoration: var(--amsterdam-link-focus-text-decoration);
-  }
-
-  &:visited {
-    color: var(--amsterdam-link-visited-color);
+    text-decoration-thickness: var(--amsterdam-link-focus-text-decoration-thickness);
```

Comment:
This says focus thickness, not visited

Reply:
It doesn't apply to the `visited` state, only to `hover`, `active` and `focus`. The standalone link doesn't have `visited` state styling.

Reply:
ah I missed the red diff